internal/provider/resource_postgresql_connection.go

```diff
@@ -0,0 +1,75 @@
+package provider
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/tfsdk"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/polytomic/polytomic-go"
+)
+
+var postgresqlResourceSchema = tfsdk.Schema{
+	MarkdownDescription: "Postgresql Connection",
+	Attributes: map[string]tfsdk.Attribute{
+		"organization": {
+			MarkdownDescription: "Organization ID",
+			Optional:            true,
+			Type:                types.StringType,
+			PlanModifiers: tfsdk.AttributePlanModifiers{
+				resource.RequiresReplace(),
+			},
+		},
+		"name": {
+			Type:     types.StringType,
+			Required: true,
+		},
+		"configuration": {
+			Attributes: tfsdk.SingleNestedAttributes(map[string]tfsdk.Attribute{
+				"hostname": {
+					Type:     types.StringType,
+					Required: true,
+				},
+				"username": {
+					Type:     types.StringType,
+					Required: true,
+				},
+				"password": {
+					Type:     types.StringType,
+					Required: true,
+				},
+				"database": {
+					Type:     types.StringType,
+					Required: true,
+				},
+				"port": {
+					Type:     types.Int64Type,
+					Required: true,
+				},
+				"ssl": {
+					Type:     types.BoolType,
+					Optional: true,
+				},
```

Comment:
Probably not necessary just yet but it would be nice to have access to the other Postgres configuration values such as SSH tunneling and client certificate auth